mdsuite/transformations/map_molecules.py

```diff
@@ -59,37 +61,34 @@ class MolecularMap(Transformations):
     """
 
     def __init__(self):
-        """Constructor for the MolecularMap class.
-
-        Parameters
-        ----------
-        molecules : dict
-                Molecule dictionary to use as reference. e.g, the input for
-                emim-PF6 ionic liquid would be.
-
-                .. code-block::
-
-                   {'emim': {'smiles': 'CCN1C=C[N+](+C1)C', 'amount': 20},
-                   'PF6': {'smiles': 'F[P-](F)(F)(F)(F)F', 'amount': 20}}
-
+        """
+        Constructor for the MolecularMap class.
         """
         super().__init__()
-        self.molecules = None
+        self.molecules = None  # parsed by the user.
         self.reference_molecules = {}
         self.adjacency_graphs = {}
-        self.dependency = mdsuite_properties.unwrapped_positions
+        self.mapping_property = "Unwrapped_Positions"
```

Comment:
is this available as `mdsuite_properties` ?

Reply:
oh yes, sorry I meant to change that before making it read for review.

Reply:
resolved in a3af13d